src/caesarCypher.js

```diff
@@ -0,0 +1,17 @@
+function upperCase(str) {
+  return str === str.toUpperCase();
+}
+
+const caesarCypher = (string, num) => {
+  let result = [];
+  for (let i = 0; i < string.length; i += 1) {
+    if (upperCase(string[i])) {
+      result += String.fromCharCode(((string.charCodeAt(i) + num - 65) % 26) + 65);
+    } else {
+      result += String.fromCharCode(((string.charCodeAt(i) + num - 97) % 26) + 97);
+    }
+  }
+  return result;
+};
```

Comment:
The basic Caesar Cipher actually interact with the characters from [A-Z] &[a-z]. For example, if I perform an operation on the '("ATTACK.AT.DAWN",5)' should return me "FYYFHP.FY.IFBS". Right now the formation is returning me 'FYYFHP3FY3IFBS'

Reply:
Fix function